int32 num_dimensions
float64[] start
float64[] goal
float64 search_radius
float64 clearance
float64 max_time
int32 max_iterations